bindings/wallet-jni/Cargo.toml

```diff
@@ -7,7 +7,7 @@ license = "MIT OR Apache-2.0"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 [lib]
-crate-type = [ "cdylib" ]
+crate-type = [ "cdylib", "staticlib" ]
```

Comment:
Apparently `cdylib` are deprecated for aarch64 builds, I added `staticlib` but I'm not sure If we need to remove the `cdylib` one.

Reply:
Hmmm, things won't work if we remove the `cdylib` though, as we are actually using the `.so` file. 

I'm not sure if we can load static libraries in jni directly, so maybe we need to tweak something. I'll try to find out.